devenv/public_cluster_tf/main.tf

```diff
@@ -75,6 +75,11 @@ data "azurerm_user_assigned_identity" "clusteridentity" {
   resource_group_name = azurerm_kubernetes_cluster.cluster.node_resource_group
 }
 
+data "azurerm_resources" "noderesourcegroup" {
```

Comment:
how is this used?

Reply:
it's only used in private - looks like it was left over from the divergence. thanks for catching that!